Permanent/ViewControllers/WelcomePageViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  WelcomePageViewController.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 06.07.2021.
+//
+
+import UIKit
+
+class WelcomePageViewController: BaseViewController<AuthViewModel>  {
```

Comment:
It might be enough to inherit from UIViewController, since this controller does not have a view model associated with it. And it doesn't have any business logic associated